ultralytics/yolo/engine/exporter.py

```diff
@@ -447,7 +447,7 @@ def export_engine(self, prefix=colorstr('TensorRT:')):
                 check_requirements('nvidia-tensorrt', cmds='-U --index-url https://pypi.ngc.nvidia.com')
             import tensorrt as trt  # noqa
 
-        check_version(trt.__version__, '7.0.0', hard=True)  # require tensorrt>=8.0.0
+        check_version(trt.__version__, '7.0.0', hard=True)  # require tensorrt>=7.0.0
```

Comment:
@AyushExel Is the lack of actual version bump here correct?

Reply:
Yup. It should be similar to https://github.com/ultralytics/ultralytics/pull/3423/files